db/gorm.go

```diff
@@ -9,7 +9,32 @@ import (
 	"strings"
 )
 
-var safeNameRE = regexp.MustCompile("^[a-zA-Z0-9_]+$")
+// Characters allowed in an unquoted identifier by MariaDB.
+// https://mariadb.com/kb/en/identifier-names/#unquoted
+var mariaDBUnquotedIdentifierRE = regexp.MustCompile("^[0-9a-zA-Z$_]+$")
```

Comment:
I believe the following should get us both the ASCII and Extended characters from the above link:

```suggestion
var mariaDBUnquotedIdentifierRE = regexp.MustCompile("^[$\w\x{0080}-\x{FFFF}]+$")
```

Reply:
Thanks! I'll skip this since I know our table names and column names will only ever be ASCII.